### ISSUE TITLE
jade / jade-brunch dependency problem

### ISSUE THREAD

**Issue by jareiko** [scotch/angular-brunch-seed#32]
When building a fresh install of angular-brunch-seed I encountered this error:

```
$ brunch build
27 Nov 19:57:58 - error: Reading of 'vendor/scripts/brunch_JavaScriptCompiler_runtime.js' failed. ENOENT, open '/Users/jareiko/projects/tr7-angular/node_modules/jade-brunch/node_modules/jade/runtime.js'
```

It may be an error in brunch itself: it looks for `jade` inside `jade-brunch`, while in fact it's already in the root `node_modules` directory so npm didn't include another copy inside `jade-brunch`.

Anyway, given that theory, I worked around the problem as follows:

```
$ npm remove jade
$ npm install jade-brunch
$ npm install
```

After this, `brunch build` succeeds.

**Comment by kylefinley** [comment]
@jareiko, 

Thank you for reporting this. I'm not entirely sure why this is occurring. As a temporary fix I have removed `jade` from the `devDependencies` -- which seems to resolve the issue. e9b887bed8aa86741d92de5da7eb8717fb7ebaa9

I'll leave this issue open until a better solution is found. 

Thanks again,

Kyle

**Comment by paulmillr** [comment]
yea brunch probably needs to load only modules which are brunch plugins, will need to fix this

**Comment by paulmillr** [comment]
fixed in jade-brunch 1.4.1

**Comment by kylefinley** [comment]
Thanks @paulmillr 

resolved with 6d463acbc3551067b6ba0e52b0a8778aeb8b3115
